cpp-client/deephaven/client/src/highlevel/impl/table_handle_impl.cc

```diff
@@ -65,11 +65,14 @@ using io::deephaven::barrage::flatbuf::ColumnConversionMode;
 using io::deephaven::barrage::flatbuf::CreateBarrageMessageWrapper;
 using io::deephaven::barrage::flatbuf::CreateBarrageSubscriptionOptions;
 using io::deephaven::barrage::flatbuf::CreateBarrageSubscriptionRequest;
-using io::deephaven::barrage::flatbuf::GetBarrageMessageWrapper;
 
 namespace deephaven::client::highlevel {
 namespace impl {
 
+inline const io::deephaven::barrage::flatbuf::BarrageMessageWrapper *GetBarrageMessageWrapper(const void *buf) {
```

Comment:
Super-nit: as a style thing I avoid the `inline` keyword unless it's required (like in a .h file where a function definition would be multiply included). Here it feels a little noisy. The compiler can choose to inline this function or not, and if not, then it's not worth much as a compiler.

Up to you.

Reply:
A bit confused why this is twice, but see above.